gcp/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -2,109 +2,48 @@ terraform {
   backend "gcs" {}
 }
 
-provider "google-beta" {
+provider "google" {
   project     = "${var.project_id}"
   credentials = "${var.serviceaccount_key}"
 }
 
-variable "nonce" {}
-variable "domain_name" {}
-variable "env" {}
-variable "organization_id" {}
-variable "project_id" {}
-variable "serviceaccount_key" {}
-variable "auth_user_email" {}
-
-# Terragrunt variables
-
-variable "notification_email" {}
-
-variable "use_auth_user_email" {
-  default = false
-}
-
-# Enables debug mode when TF_VAR_stackdriver_debug is not empty
-
-variable "stackdriver_debug" {
-  default = ""
+provider "google-beta" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
 }
 
-resource "template_dir" "resources" {
-  source_dir      = "${path.cwd}/resources"
-  destination_dir = "${path.cwd}/resources_rendered"
-
-  vars {
-    project_id         = "${var.project_id}"
-    organization_id    = "${var.organization_id}"
-    domain_name        = "${var.domain_name}"
-    notification_email = "${(var.use_auth_user_email && var.auth_user_email != "") ? var.auth_user_email : var.notification_email}"
-
-    enabled = "${(var.env == "prd" || var.env == "stg") ? true : false}"
-  }
+variable "crm_sa" {
+  default = "one-platform-tenant-manager@system.gserviceaccount.com"
 }
 
-resource "null_resource" "apply_stackdriver_monitoring" {
-  depends_on = ["template_dir.resources"]
+variable "project_id" {}
+variable "organization_id" {}
+variable "notification_email" {}
+variable "common_project_id" {}
+variable "domain_name" {}
+variable "serviceaccount_key" {}
+variable "env" {}
 
+resource "null_resource" "destroy_old_stackdriver_resources" {
+  # this script is only needed to run once to migrate the resources from json to TF
   triggers = {
-    nonce = "${var.nonce}"
+    nonce = "1"
```

Comment:
Looks like the triggers section is not needed at all?

Reply:
removed